src/main/java/com/twilio/twiml/voice/Refer.java

```diff
@@ -109,11 +109,21 @@ public Builder method(HttpMethod method) {
         /**
          * Add a child {@code <Sip>} element
          */
-        public Builder referSip(ReferSip referSip) {
+        public Builder sip(ReferSip referSip) {
             this.children.add(referSip);
             return this;
         }
 
+        /**
+         * @deprecated
+         * This method is deprecated. Use {@link #sip(ReferSip)}.
+         * Add a child {@code <Sip>} element
+         */
```

Comment:
Prefer the description come first.

Also, elsewhere in this lib it's like this: `@deprecated use {@link #sip(ReferSip)} instead.`